src/xcode/ENA/ENA/Source/Scenes/TraceLocations/Details/TraceLocationDetailsViewController.swift

```diff
@@ -74,9 +82,8 @@ class TraceLocationDetailsViewController: UIViewController, FooterViewHandling {
 					}
 				}
 			case let .failure(error):
-					self?.footerView?.setLoadingIndicator(false, disable: false, button: .primary)
-				
-				Log.error("Could not retrieve QR code poster template from protobuf.", log: .qrCode, error: error)
+				self?.footerView?.setLoadingIndicator(false, disable: false, button: .primary)
```

Comment:
You want to set the same state in both cases, why not set it before the switch?
The return in ..failure(error) line 87 seems to be obsolet as well

Reply:
Fixed